js/client-mainmenu.js

```diff
@@ -114,7 +125,68 @@
 				);
 			}
 		},
+		createLoginFormObserver:function (self) {
+			const config = {attributes: true, childList: true, subtree: true};
+			const observer = new MutationObserver(function () {
```

Comment:
Why is this necessary? Can you not just use jQuery?

Reply:
Yeah, but I'm asking if you can't just use jQuery to watch the DOM instead? Or why not set a listener trigger in the code that handles completing logins?